### PR TITLE
refactor: remove fragile selectors

### PR DESCRIPTION
**Pull request by rabi-siddique**
The PR does the following:
- Eliminates fragile selectors used in e2e test cases. 
- Increase timeouts for some tests to ensure stability with emerynet.

### CODE REVIEW

File: test/e2e/specs/test.spec.js [hunk 1]
```diff
@@ -102,20 +102,27 @@ describe('Wallet App Test Cases', { execTimeout: DEFAULT_TIMEOUT }, () => {
...
-          fromAddress: accountAddresses.user2,
-          giveAmount: '2IST',
-          discount: 5,
-        }).then(() => {
-          cy.getTokenAmount('IST').then((tokenValue) => {
-            expect(tokenValue).to.lessThan(initialTokenValue);
+    it(
+      'should place a bid by discount from the CLI successfully and verify IST balance',
+      {
+        defaultCommandTimeout: DEFAULT_TIMEOUT,
```
Comment (turadg) [inline test/e2e/specs/test.spec.js:108]: Is there a way to set these defaults globally?

File: test/e2e/test.utils.js [hunk 2]
```diff
@@ -15,7 +15,8 @@ export const accountAddresses = {
 };
 
 export const EMERYNET_FAUCET_URL = 'https://emerynet.faucet.agoric.net';
-export const DEFAULT_TIMEOUT = 2 * 60 * 1000;
+export const DEFAULT_TIMEOUT = 3 * 60 * 1000;
+export const DEFAULT_TASK_TIMEOUT = 3 * 60 * 1000;
```
Comment (turadg) [inline test/e2e/test.utils.js:19]: Curious why distinguishing these now, given they're the same value.

File: test/e2e/specs/test.spec.js (hunk 1 above)
Comment (rabi-siddique) [reply test/e2e/specs/test.spec.js:108]: We can set these globally in `synpress.config.cjs`. Do you think it's a good idea to set them globally? Currently, we are only using these defaults in tests where they are specifically required

File: test/e2e/test.utils.js (hunk 2 above)
Comment (rabi-siddique) [reply test/e2e/test.utils.js:19]: It's mainly to ensure consistency and stability across our tests. The `DEFAULT_TIMEOUT` is the timeout for actions involving waiting for DOM elements. The `DEFAULT_TASK_TIMEOUT` applies to custom commands created in `support.js`.

In many tests, we use a mix of both(Wait for DOM elements as well as running custom commands). So increasing the timeout for both, we can achieve more consistent and stable test results. I've implemented similar timeouts in `dapp-inter` for liquidation tests, and they have been very stable with emerynet.

Also, thanks for asking. I just noticed that the tests using both of these timeouts actually need to use `DEFAULT_TASK_TIMEOUT`.

File: test/e2e/specs/test.spec.js (hunk 1 above)
Comment (turadg) [reply test/e2e/specs/test.spec.js:108]: well the values are called `DEFAULT_TIMEOUT` and `DEFAULT_TASK_TIMEOUT`. If they're not really the default values, then please change the name. E.g. `SLOW_TIMEOUT`. Then update the config with the actual desired defaults.

File: test/e2e/test.utils.js (hunk 2 above)
Comment (turadg) [reply test/e2e/test.utils.js:19]: Helpful explanation. Please include it in comments in the code.

Comment (rabi-siddique) [reply test/e2e/test.utils.js:27]: Updated :)

**Pull request merged**